From a6747d9fad7d15345f7703e9f9d2916ea20a6edf Mon Sep 17 00:00:00 2001
From: Phil Elwell <phil@raspberrypi.com>
Date: Fri, 25 Feb 2022 09:53:56 +0000
Subject: [PATCH] overlays: Add "drm" parameter to pitft35-resistive

The "drm" parameter forces the use of the hx8357d DRM driver, when by
default the fb_hx8357d framebuffer driver will be used in preference.

See: https://forums.raspberrypi.com/viewtopic.php?t=330088

Signed-off-by: Phil Elwell <phil@raspberrypi.com>
---
 arch/arm/boot/dts/overlays/README                        | 4 ++++
 arch/arm/boot/dts/overlays/pitft35-resistive-overlay.dts | 1 +
 2 files changed, 5 insertions(+)

--- a/arch/arm/boot/dts/overlays/README
+++ b/arch/arm/boot/dts/overlays/README
@@ -2689,6 +2689,10 @@ Params: speed                   Display
 
         debug                   Debug output level {0-7}
 
+        drm                     Force the use of the hx8357d DRM driver (by
+                                default the fb_hx8357d framebuffer driver will
+                                be used in preference if available)
+
 
 Name:   pps-gpio
 Info:   Configures the pps-gpio (pulse-per-second time signal via GPIO).
--- a/arch/arm/boot/dts/overlays/pitft35-resistive-overlay.dts
+++ b/arch/arm/boot/dts/overlays/pitft35-resistive-overlay.dts
@@ -115,5 +115,6 @@
 		rotate =  <&pitft>,"rotate:0";
 		fps =     <&pitft>,"fps:0";
 		debug =   <&pitft>,"debug:0";
+		drm =     <&pitft>,"compatible=adafruit,yx350hv15";
 	};
 };
